crates/ide/src/hover.rs

```diff
@@ -67,12 +71,12 @@ pub struct HoverResult {
     pub actions: Vec<HoverAction>,
 }
 
-// Feature: Hover
-//
-// Shows additional information, like type of an expression or documentation for definition when "focusing" code.
-// Focusing is usually hovering with a mouse, but can also be triggered with a shortcut.
-//
-// image::https://user-images.githubusercontent.com/48062697/113020658-b5f98b80-917a-11eb-9f88-3dbc27320c95.gif[]
+/// Feature: Hover
+///
+/// Shows additional information, like the type of an expression or the documentation for a definition when "focusing" code.
+/// Focusing is usually hovering with a mouse, but can also be triggered with a shortcut.
+///
+/// image::https://user-images.githubusercontent.com/48062697/113020658-b5f98b80-917a-11eb-9f88-3dbc27320c95.gif
```

Comment:
This needs to use only two slashes, otherwise it won't be picked up by our docgen infra.